Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.xaml

```diff
@@ -12,7 +12,7 @@
     <Thickness x:Key="TokenizingTextBoxPresenterMargin">0,0,6,0</Thickness>
     <x:Double x:Key="TokenizingTextBoxTokenSpacing">2</x:Double>
     <Thickness x:Key="TextControlBorderThemeThicknessFocused">2</Thickness>
-
+    
```

Comment:
was this intentional ?

Reply:
I had something here and then moved it in a separate commit as it was in the wrong place, so guess the whitespace changed. between those (probably 4 spaces in one and blank in the other).